phi/tools/linear_tools.py

```diff
@@ -0,0 +1,254 @@
+import requests
+from os import getenv
+from phi.tools import Toolkit
+from phi.utils.log import logger
+
+
+class LinearTool(Toolkit):
+    def __init__(self):
+        super().__init__(name="linear tools")
+        self.api_token = getenv("LINEAR_API_KEY")
+        self.endpoint = "https://api.linear.app/graphql"
+        self.headers = {"Authorization": f"Bearer {self.api_token}"}
```

Comment:
You haven't registered the functions. Take example from https://github.com/phidatahq/phidata/blob/main/phi/tools/slack.py